pandas/tseries/index.py

```diff
@@ -1256,6 +1256,19 @@ def get_loc(self, key):
             except (KeyError, ValueError):
                 raise KeyError(key)
 
+    def get_loc_nearest(self, key, side='nearest'):
```

Comment:
These should also get a docstring (there are normally some decorators for adding a docstring)